src/__tests__/expected/selectTwoCommandMode.txt

```diff
@@ -0,0 +1,30 @@
+
+
+
+
+
+
+
+
+==========================================================
+Files you have selected:
+==========================================================
+/foo/bar/README.md
```

Comment:
alright here we go. see! this is what a user would see as if they were in the UI